eng/scripts/Invoke-GeneratAndBuildV2.ps1

```diff
@@ -0,0 +1,87 @@
+#Requires -Version 7.0
+param (
+  [string]$inputJsonFile,
+  [string]$outputJsonFile
+)
+
+. (Join-Path $PSScriptRoot automation GenerateAndBuildLib.ps1)
+
+$inputJson = Get-Content $inputJsonFile | Out-String | ConvertFrom-Json
+$swaggerDir = $inputJson.specFolder
+$swaggerDir = Resolve-Path $swaggerDir
+$swaggerDir = $swaggerDir -replace "\\", "/"
+[string[]] $inputFilePaths = $inputJson.changedFiles;
+$readmeFiles = $inputJson.relatedReadmeMdFiles
+$commitid = $inputJson.headSha
+$repoHttpsUrl = $inputJson.repoHttpsUrl
+$downloadUrlPrefix = $inputJson.installInstructionInput.downloadUrlPrefix
+[string] $autorestConfig = $inputJson.autorestConfig
+
+$autorestConfigYaml = ""
+if ( $null -ne $autorestConfig -and $autorestConfig -ne "") {
+    $autorestConfig | Set-Content "config.md"
+    $autorestConfigYaml = Get-Content -Path .\config.md
+    $range = ($autorestConfigYaml | Select-String -Pattern '```').LineNumber
+    if ( $range.count -gt 1) {
+        $startNum = $range[0];
+        $lines = $range[1] - $range[0] - 1
+        $autorestConfigYaml = ($autorestConfigYaml | Select -Skip $startNum | Select -First $lines) |Out-String
```

Comment:
When you call `Get-Content` without `-Raw` it returns an array type (the file split on newline). So this variable may be a string or an array depending on conditions which could cause unintended behavior.

Reply:
Thanks @benbp. I need to read this content as string array.